client/src/components/Collections/common/CollectionEditView.vue

```diff
@@ -0,0 +1,225 @@
+<template>
+    <div>
+        <h4>{{ l("Edit Collection Attributes") }}</h4>
+        <b-alert show variant="info" dismissible>
+            {{ l("This will create a new collection in your History. Your quota usage will not increase. ") }}
+        </b-alert>
+        <div v-if="jobError">
+            <b-alert show variant="danger" dismissible>
+                {{ errorMessage }}
+            </b-alert>
+        </div>
+        <b-tabs content-class="mt-3">
+            <b-tab>
+                <template v-slot:title> <font-awesome-icon icon="table" /> &nbsp; {{ l("Database/Build") }}</template>
+                <database-edit-tab
+                    :databaseKeyFromElements="databaseKeyFromElements"
+                    :genomes="genomes"
+                    @clicked-save="clickedSave"
+                />
+            </b-tab>
+            <b-tab v-if="atleastOneSuitableConverter">
+                <template v-slot:title> <font-awesome-icon icon="cog" /> &nbsp; {{ l("Convert") }}</template>
+                <div class="alert alert-secondary" role="alert">
+                    <div class="float-left">Convert all datasets to new format</div>
+                    <div class="text-right">
+                        <button
+                            class="run-tool-collection-edit btn btn-primary"
+                            @click="clickedConvert"
+                            :disabled="chosenConverter == {}"
+                        >
+                            {{ l("Convert Collection") }}
+                        </button>
+                    </div>
+                </div>
+                <b>{{ l("Convterter Tool: ") }}</b>
+                <multiselect
+                    v-model="chosenConverter"
+                    deselect-label="Can't remove this value"
+                    track-by="name"
+                    label="name"
+                    :options="suitableConverters"
+                    :searchable="true"
+                    :allow-empty="true"
+                >
+                </multiselect>
+            </b-tab>
+            <b-tab>
+                <template v-slot:title> <font-awesome-icon icon="database" /> &nbsp; {{ l("Datatype") }}</template>
```

Comment:
Want to comment this out for this PR like we had discussed?

Reply:
Done